tests/multinode.rs

```diff
@@ -813,6 +813,7 @@ fn test_multi_node_dynamic_network() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Just pushed a change that should fix these tests, can you give them another try?